todo_app/app.py

```diff
@@ -1,39 +1,96 @@
 from flask import Flask, render_template, request, redirect, url_for
+from flask_login import LoginManager, login_required, login_user, UserMixin, current_user
 from todo_app.data.trello_items import get_items, add_item, item_in_progress, item_completed, reset_item_status
-
+from todo_app.data.user_login import get_user_identity_endpoint, get_user_data_endpoint, get_access_token_endpoint
+import os
 from todo_app.flask_config import Config
 from todo_app.view_model import ViewModel
+from functools import wraps
+
+def authorised_user(func):
```

Comment:
You've defined this function, but you're not using it anywhere! I'm guessing this happened when converting to a decorator, but this lets me bypass your security since it's only on the client/HTML side; if I can guess/find your relevant endpoints then there's nothing to stop me hitting them.

We should make sure we're using this in appropriate functions, e.g.
```
    @app.route('/items/new', methods=['POST'])
    @login_required
    @authorised_user
    def add_new_item():
```